docs updates (#112)
skorokithakis/django-loginas: README.md

privilege escalation from staff users to superusers. If you want to log in as
a superuser, first demote them to a non-superuser, and then log in.

### URL Issues

> After loginas a user, if you are getting logged out entirely, then switch to:

Reviewer: ```suggestion
If you are getting logged out entirely after you log in as a user, switch to:
```
